schemas/iso19139/src/main/plugin/iso19139/present/metadata-fop.xsl

```diff
@@ -159,13 +159,13 @@
     </xsl:call-template>
 
     <!-- Charset Encoding -->
-    <xsl:variable name="lang">
+    <xsl:variable name="characterSet">
```

Comment:
Do you have a bug related to this file ? I don't think the UI is using it anymore unless you are using old Jeeves PDF printing service ?

Reply:
I was using intellij and it reported a warning indicating that there was duplicate variables.  I'm not aware of any bug that this caused - I just fixed it because it was obvious bug and could have caused issues...